src/contexts/WalletConnectionProvider/HardcodedWalletStandardAdapter.tsx

```diff
@@ -1,27 +1,45 @@
-import { BaseSignerWalletAdapter, WalletName, WalletNotConnectedError, WalletReadyState, isVersionedTransaction } from "@solana/wallet-adapter-base";
-import { Keypair, Transaction, TransactionVersion, VersionedTransaction } from "@solana/web3.js";
+import {
+  BaseSignerWalletAdapter,
+  WalletName,
+  WalletNotConnectedError,
+  WalletReadyState,
+  isVersionedTransaction,
+} from '@solana/wallet-adapter-base';
+import { Keypair, Transaction, TransactionVersion, VersionedTransaction } from '@solana/web3.js';
+import { isIosAndRedirectable } from 'src/misc/utils';
```

Comment:
Using absolute paths will lead to the `Could not resolve "src/misc/utils"` error

```suggestion
import { isIosAndRedirectable } from '../../misc/utils';
```